packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -143,13 +145,28 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
         const routeSchemas = routeSchemasFromRouteConfig(route, method);
 
         this.routes.push({
-          handler: async (req, responseToolkit) =>
-            await this.handle({
+          handler: async (req, responseToolkit) => {
+            const startUtilization = performance.eventLoopUtilization();
+
+            const response = await this.handle({
               routeSchemas,
               request: req,
               responseToolkit,
               handler: this.enhanceWithContext(handler),
-            }),
+            });
+
+            const eventLoopUtilization = performance.eventLoopUtilization(
+              performance.eventLoopUtilization(),
+              startUtilization
+            );
+
+            apm.currentTransaction?.addLabels({
+              event_loop_utilization: eventLoopUtilization.utilization,
+              event_loop_active: eventLoopUtilization.active,
+            });
```

Comment:
Are these labels correctly mapped later on so we can run numeric aggregations on them?

Reply:
That automatically happens: https://www.elastic.co/guide/en/apm/guide/current/data-model-metadata.html